hice_backend/urls.py

```diff
@@ -158,4 +158,9 @@
         CreateRecordingView.as_view(),
         name="create-recording",
     ),
+    path(
+        "quizzes/<uuid:recording_id>/",
```

Comment:
I don't think this is a good pattern; how about we do:

`recordings/<uuid:recording_id>/quizzes/`

We need to do some refactoring at some point to standardize `recordings` and `instructor-recordings`